src/main/java/com/deloitte/elrr/datasync/scheduler/CreateUpdateImport.java

```diff
@@ -0,0 +1,97 @@
+package com.deloitte.elrr.datasync.scheduler;
+
+import java.sql.Timestamp;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+import org.springframework.transaction.annotation.Propagation;
+import org.springframework.transaction.annotation.Transactional;
+
+import com.deloitte.elrr.datasync.entity.Import;
+import com.deloitte.elrr.datasync.exception.ResourceNotFoundException;
+import com.deloitte.elrr.datasync.jpa.service.ImportService;
+
+import lombok.extern.slf4j.Slf4j;
+
+@Component
+@Slf4j
+public class CreateUpdateImport {
+
+    @Autowired
+    private ImportService importService;
+
+    @Value("${initial.date}")
+    private Timestamp initialDate;
+
+    /**
+     * @return importRecord
+     */
+    @Transactional(propagation = Propagation.REQUIRES_NEW)
```

Comment:
Are you trying to escape the transaction scope with this so that the next run sees the change reflected before txn commit?